helm/dinghy-ping/templates/rbac.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "dinghy-ping.fullname" . }}
```

Comment:
I believe we need namespace here

Reply:
`namespace: default`
Or, however we're templating it

Reply:
Looks like `{{ .Release.Namespace }}` will work here cool!